tekton/README.md

```diff
@@ -1,60 +1,61 @@
 
-This directory contains the Tekton `Tasks` and `Pipelines` used to create Dashboard releases. 
+This directory contains the Tekton `Tasks` and `Pipelines` used to create Dashboard releases.
 
-These tasks run on your local cluster, and then copy the release artifacts - docker images and yaml files - into [the `tekton releases` bucket in Google Cloud Platform](https://console.cloud.google.com/storage/browser/tekton-releases/dashboard). Your cluster much contain keys from rom a google account with the necessary authorization in order for this to work. 
+These tasks run on your local cluster, and then copy the release artifacts - docker images and yaml files - into [the `tekton releases` bucket in Google Cloud Platform](https://console.cloud.google.com/storage/browser/tekton-releases/dashboard). Your cluster much contain keys from rom a google account with the necessary authorization in order for this to work.
 
 ## Release process
 
 First, ensure that your credentials are set up correctly. You will need an account with access to [Google Cloud Platform](https://console.cloud.google.com). Your account must have 'proper authorization to release the images and yamls' in the [`tekton-releases` GCP project](https://github.com/tektoncd/plumbing#prow). Your account must have `Permission iam.serviceAccountKeys.create`. Contact @bobcatfish or @dlorenc if you are going to be creating dashboard releases and require this authorization.
 
-- You will need to install the [`gcloud` CLI](https://cloud.google.com/sdk/gcloud/) in order to get keys out of Google Cloud and into your local cluster. These credentials will be patched onto the service account to be used by the Tekton PipelineRuns. You do not need to create a new GCP project or pay Google any money. 
-- It's convenient to use the ['tkn' CLI](https://github.com/tektoncd/cli) to kick off builds, so grab that as well. 
+- You will need to install the [`gcloud` CLI](https://cloud.google.com/sdk/gcloud/) in order to get keys out of Google Cloud and into your local cluster. These credentials will be patched onto the service account to be used by the Tekton PipelineRuns. You do not need to create a new GCP project or pay Google any money.
+- It's convenient to use the ['tkn' CLI](https://github.com/tektoncd/cli) to kick off builds, so grab that as well.
 
 ```bash
 KEY_FILE=release.json
 GENERIC_SECRET=release-secret
-# The kubernetes ServiceAccount that will be used by your Tekton tasks. 'default' is the default. It should all ready exist. 
-SERVICE_ACCOUNT=default 
+# The kubernetes ServiceAccount that will be used by your Tekton tasks. 'default' is the default. It should all ready exist.
+SERVICE_ACCOUNT=default
 GCP_ACCOUNT="release-right-meow@tekton-releases.iam.gserviceaccount.com"
 
-# 1. Create a GCP private key for the service account. You may want to keep a copy somewhere safe. 
+# 1. Create a GCP private key for the service account. You may want to keep a copy somewhere safe.
```

Comment:
While you're here... let's say `definitely` keep it somewhere safe, lest you enter the "ran out of keys" problem

Reply:
done